pkg/logwrite/logwrite.go

```diff
@@ -62,8 +73,8 @@ func NewLogFile(dir, name string) (*LogFile, error) {
 }
 
 // Write appends a message to the log file
-func (l *LogFile) Write(m *LogMessage) error {
-	s := m.String() + "\n"
+func (l *LogFile) Write(m *LogMessage, formatLog string) error {
+	s := m.String(formatLog) + "\n"
```

Comment:
maybe use the same variable everywhere? `logFormat`.

Reply:
Sure, my bad

Reply:
fixed